src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -120,6 +138,40 @@ class MetaInfoPanelImpl extends React.PureComponent<Props> {
     }
   }
 
+  _handleMoreInfoButtonClick = () => {
+    this.setState((state) => ({ showsMoreInfo: !state.showsMoreInfo }));
+  };
+
+  _renderMoreInfoSection(section: ExtraProfileInfoSection) {
+    return (
+      <div key={section.label}>
+        <h2 className="metaInfoSubTitle" key={'title ' + section.label}>
```

Comment:
you don't need a `key` attribute here, they're necessary only when rendering arrays.

(So it's good to have the `key` for the div above)

Reply:
I don't know why I put them there...